src/Makevars.win

```diff
@@ -0,0 +1,5 @@
+PKG_CFLAGS = $(SHLIB_OPENMP_CFLAGS)
```

Comment:
`configure` will only run on non-Windows, which means that `src/Makevars` will not be generated there, so we need to supply `Makevars.win` for windows. But we don't really need to check if OpenMP is supported, because the Rtools compilers all support it.